main.py

```diff
@@ -902,6 +982,26 @@ def handle_interface_link_up(self, interface, event):
             self._intfs_updated_at[interface.id] = event.timestamp
         self.handle_link_up(interface)
 
+    def _notify_link_from_interface(self, interface):
+        """Determines whether to send a notification."""
+        link = interface.link
+        if interface.id == link.endpoint_a.id:
+            other_intf = link.endpoint_b
+        else:
+            other_intf = link.endpoint_a
+        if other_intf.is_enabled() and not interface.is_enabled():
+            self.notify_link_enabled_state(link, "enabled")
+
+    def get_flows_by_switch(self, dpid) -> dict:
+        """Get installed flows by switch from flow_manager."""
+        endpoint = settings.FLOW_MANAGER_URL +\
+            f'/stored_flows?state=installed&dpid={dpid}'
+        response = httpx.get(endpoint)
+        if response.status_code != 200:
+            detail = f"Error while getting flows: {response.json()}."
```

Comment:
It's safer to use `response.text` instead. Right we do have a middleware ensuring that any `HTTPException` will be returned as json, but still other non handled exceptions will either return their traceback or a generic internal error string if the traceback config is false on KytosConfig.

Reply:
Changed, although now the description has backslashes `"Error while getting flows: {\"description\":\"Error\",\"code\":400}."`

Reply:
Right. Yes, that's a bit awkward for 4XX errors, when handling `res.status_code in (404, 400)` it's safe to try to get `response.json()`, but for certain 500s they're not. If you wish to improve and isolate them and only include `response.text` that's OK too. Or just keeping `response.text` for everything here it's OK too, backslashes don't look pretty but we know it won't crash at least. It's up to you. On `telemetry_int` client, [I ended up using just `response.text`](https://github.com/kytos-ng/telemetry_int/blob/master/kytos_api_helper.py#L115)

Reply:
Ok, going to leave it as it is.